Source/Initialization/WarpXInitData.cpp

```diff
@@ -74,6 +74,9 @@ WarpX::InitData ()
         if (plot_int > 0)
             WritePlotFile();
 
+        if (openpmd_int > 0)
```

Comment:
Hm, adding this seams to cause a hang in `bp` dumps...

Reply:
Potentially fixed in 0.11.0-dev of openPMD-api via https://github.com/openPMD/openPMD-api/pull/674